s3/replication/manager/src/systemd/s3replicationmanager.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=s3replicationmanager service file to start a daemon instance.
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+User=root
+Restart=on-failure
+Type=simple
+ExecStart=/usr/bin/python3 -m s3replicationmanager
```

Comment:
try this option
ExecStart=${REPLICATION_VENV)/bin/python3 -m s3replicationmanager

Reply:
done. To get the Environment values from the environment or in form of command line args, there are two ways.
1. we can pass a configfile path which contains environment variables using EnvrionmentFile option. Which is an extra overhead.
2. There's another way to pass command line arguments by keeping a service file name of type my_svc_file@.service. This option is exclusive for command line options to the ExecStart command in service file. so this is of no use.
As this is a single line change, I have kept it simple, by populating ExecStart command from build_and_test.sh script only.